game/MainScreen.gd

```diff
@@ -1,15 +1,100 @@
 extends Control
 
+onready var indicator_active = preload("res://UI_Elements/complete progress indicator.png")
+onready var indicator_inactive = preload("res://UI_Elements/empty progress indicator.png")
+
+
+export (NodePath) onready var back_step = get_node(back_step) as TextureButton
+export (NodePath) onready var next_step = get_node(next_step) as TextureButton
+export (NodePath) onready var steps_container =  get_node(steps_container) as HBoxContainer
 export (NodePath) onready var card_button = get_node(card_button) as TextureButton
 export (NodePath) onready var character_sheet_panel = get_node(character_sheet_panel) as Control
 export (NodePath) onready var ethnicity_element = get_node(ethnicity_element) as Control
+export (NodePath) onready var profession_element = get_node(profession_element) as Control
+export (NodePath) onready var ethnicity_button = get_node(ethnicity_button) as TextureButton
+export (NodePath) onready var profession_button = get_node(profession_button) as TextureButton
+export (NodePath) onready var attributes_button = get_node(attributes_button) as TextureButton
+export (NodePath) onready var specjalization_button = get_node(specjalization_button) as TextureButton
+export (NodePath) onready var skills_button = get_node(skills_button) as TextureButton
+export (NodePath) onready var tricks_button = get_node(tricks_button) as TextureButton
+export (NodePath) onready var diseases_button = get_node(diseases_button) as TextureButton
+export (NodePath) onready var reputation_button = get_node(reputation_button) as TextureButton
+export (NodePath) onready var form_button = get_node(form_button) as TextureButton
+export (NodePath) onready var gear_button = get_node(gear_button) as TextureButton
+export (NodePath) onready var choose_profession_step = get_node(choose_profession_step) as Control
 
 
-func _ready():
-	pass
-	
+var current_step = 0
+onready var steps = [
+					ethnicity_element
+					,profession_element
+]
+
+onready var buttons = [
+					ethnicity_button
+					,profession_button
+]
+
 
 func _on_CardButton_button_up():
 	character_sheet_panel.visible = true
 	character_sheet_panel.mouse_filter = Control.MOUSE_FILTER_PASS
 	character_sheet_panel.update_card()
+
+
+func _next_step():
+	if current_step == len(steps)-1:
+		print("The End")
+	else:
+		current_step += 1
+	back_step.disabled = false
+	match current_step :
```

Comment:
This also has to be discussed - we need to clean it up a bit so it does not spaghetti out of control later.

Reply:
You fixed that :)